analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/Verifier.cs

```diff
@@ -54,83 +55,149 @@ public static void VerifyNoExceptionThrown(string path, IEnumerable<DiagnosticAn
             DiagnosticVerifier.GetAllDiagnostics(compilation, diagnosticAnalyzers, checkMode);
         }
 
-        public static void VerifyCSharpAnalyzer(string snippet,
+        public static void VerifyNonConcurrentCSharpAnalyzer(string snippet,
```

Comment:
Do we really need to rename these language-specific overloads? I'd keep the names short as they were. And maybe just add a documentation comment that it performs nonconcurrent verification.